cypress/support/elements/common/Canvas.js

```diff
@@ -53,23 +53,30 @@ class Canvas {
   getEditTitleIcon() {
     return cy.get('[data-test=personal-doc-title] [data-test=edit-icon]');
   }
-
+  // the force:true assertions on lines 63-68 are likely needed because of the
+  // Jira ticket:https://concord-consortium.atlassian.net/browse/CLUE-81
+  // Once that's fixed we can remove the force:true assertions
   createNewExtraDocumentFromFileMenu(title, type) {
     this.openFileMenu();
     cy.get('[data-test=list-item-icon-open-workspace]').click();
     cy.get('.primary-workspace .doc-tab.my-work.workspaces').click();
     cy.get('[data-test=' + type + '-section-workspaces-documents] [data-test=my-work-new-document]').click();
     dialog.getDialogTitle().should('exist').contains('Create Extra Workspace');
-    dialog.getDialogTextInput().click().clear().type(title);
+    dialog.getDialogTextInput()
+      .click({force: true})
+      .wait(100)
+      .type('{selectall}{backspace}' + title, {force: true});
```

Comment:
Oh, I just put in a similar fix in my Question tile PR.  I was wondering why random tests were starting to fail, your note about the re-styling of dialogs seems like a likely answer.

Anyway this will be a merge conflict but should be easily resolved.